src/Functional.CQS.AOP.IoC.SimpleInjector.MetricsCapturing.Tests/TypeExtensionsTests.cs

```diff
@@ -0,0 +1,120 @@
+using FluentAssertions;
+using Functional.CQS.AOP.CommonTestInfrastructure.DummyObjects;
+using Functional.CQS.AOP.CommonTestInfrastructure.MetricsCapturing.DummyObjects;
+using Functional.CQS.AOP.MetricsCapturing;
+using Functional.Primitives.FluentAssertions;
+using Xunit;
+
+namespace Functional.CQS.AOP.IoC.SimpleInjector.MetricsCapturing.Tests
+{
+	public class TypeExtensionsTests
+	{
+		[Fact]
+		public void ShouldReturnExpectedQueryAndResultTypeForDummyQueryReturnsValueTypeCachingStrategy() => VerifyQueryMetricsCapturingStrategyType<DummyQueryReturnsValueType, DummyQueryReturnsValueTypeResult, DummyQueryReturnsValueTypeMetricsCapturingStrategy>();
```

Comment:
These first four test names say `CachingStrategy` instead of `MetricsCapturingStrategy`. I assume that's a typo.

Reply:
Yes, it is indeed a typo.  Fixed.